package.json

```diff
@@ -27,6 +27,6 @@
     "grunt-contrib-watch": "latest",
     "grunt-string-replace": "^1.2.1",
     "webpack": "1.13.0",
-    "webpack-dev-server": "1.14.1"
+    "webpack-dev-server": "3.1.11"
```

Comment:
webpack-dev-server 3 doesn't support webpack < 4
Quick fix:
```
    "webpack": "4.41.5",
    "webpack-cli": "3.3.10",
    "webpack-dev-server": "3.3.1"
```

BTW, such usage is a bit weird, because webpack is not used for build at all. I suggest using something else like [node-static](https://www.npmjs.com/package/node-static) for static development server.